src/pulse3D/excel_writer.py

```diff
@@ -275,14 +275,16 @@ def write_xlsx(
     file_suffix = "full" if is_full_analysis else f"{start_time}-{end_time}"
     output_file_name = f"{input_file_name_no_ext}_{file_suffix}.xlsx"
 
-    if first_wf.stiffness_override:
-        # reverse dict to use the stiffness factor as a key and get the label value
-        post_stiffness_factor = {v: k for k, v in POST_STIFFNESS_LABEL_TO_FACTOR.items()}[
-            first_wf.stiffness_factor
-        ]
+    if not plate_recording.is_optical_recording:
```

Comment:
Should remove this check since stiffness_factor should just be set to None instead

Reply:
I think this check is necessary. stiffness_factor is already getting set to none and errors without this check.